pkg/stores/partition/store.go

```diff
@@ -44,42 +38,18 @@ type Partitioner interface {
 // Store implements types.Store for partitions.
 type Store struct {
 	Partitioner    Partitioner
-	listCache      *cache.LRUExpireCache
-	asl            accesscontrol.AccessSetLookup
 	namespaceCache corecontrollers.NamespaceCache
 }
 
 // NewStore creates a types.Store implementation with a partitioner and an LRU expiring cache for list responses.
-func NewStore(partitioner Partitioner, asl accesscontrol.AccessSetLookup, namespaceCache corecontrollers.NamespaceCache) *Store {
-	cacheSize := defaultCacheSize
-	if v := os.Getenv(cacheSizeEnv); v != "" {
-		sizeInt, err := strconv.Atoi(v)
-		if err == nil {
-			cacheSize = sizeInt
-		}
-	}
+func NewStore(partitioner Partitioner, namespaceCache corecontrollers.NamespaceCache) *Store {
 	s := &Store{
 		Partitioner:    partitioner,
-		asl:            asl,
 		namespaceCache: namespaceCache,
 	}
-	if v := os.Getenv(cacheDisableEnv); v == "false" {
```

Comment:
Does this make us ready to remove the `cacheDisableEnv` const above too?

Reply:
Yes, actually can remove all cache related consts from this package- fixed.